draft-ietf-tls-esni.md

```diff
@@ -928,6 +959,28 @@ in-band when possible, such as through the use of OCSP stapling, and clients
 SHOULD take steps to minimize or protect such requests during certificate
 validation.
 
+## Abuse of ECH Acceptance Signal
+
+To signal acceptance, the backend server overwrites 8 bytes of its
+ServerHello.random with a value provided by the client. (See
+{{backend-server-behavior}} for details.) In the presence of an active
+adversary, this behavior increases the likelihood of the ServerHello.random
+colliding with the ServerHello.random of a previous session, potentially
+reducing the overall security of the protocol. However, the remaining 24 bytes
+provide enough entropy to ensure this is not a practical avenue of attack.
+
+On the other hand, the probability that two 8-byte strings are the same is
+non-negligible. This poses a modest operational risk. Suppose the client-facing
+server terminates the connection: if the last 8 bytes of its ServerHello.random
```

Comment:
```suggestion
server terminates the connection, but does not process the ech extension: if the last 8 bytes of its ServerHello.random
```

Reply:
Changed to "i.e., ECH is rejected or ignored"